src/scripts/main.js

```diff
@@ -1,3 +1,26 @@
 'use strict';
 
-// write code here
+const tree = document.querySelector('.tree');
+
+[...document.querySelectorAll('li')].map((li) => {
```

Comment:
Consider using `forEach` instead of `map` since `map` is used for creating a new array. 🤔